migrateUpWithWrapper.mjs

```diff
@@ -0,0 +1,36 @@
+/**
+ * Hacky script that wraps the 'node-pg-migrate' program
+ * and make sure that logged errors do not leak some
+ * sensitive information such as db password for instance..
+ * */
+import dotenv from 'dotenv';
+import { spawn } from 'child_process';
+
+dotenv.config();
+
+const dbUser = process.env.PGUSER;
+const dbPassword = process.env.PGPASSWORD;
+
+const child = spawn('npm', ['run', 'migrate', 'up']);
+
+child.stdout.on('data', (data) => {
+    console.log(`${data}`);
```

Comment:
should we add the same logic in log just in case of ?

Reply:
Good comment. I thought about it too. However, I could not reproduce (I never noticed leaks while logging in stdout). Since, I cannot reproduce, I would let it that way, but keep an eye on the situation.